tests/flow/object_api/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -182,14 +182,18 @@ var sealed = { one: \"one\", two: \"two\" };
 
 // error, Array<string>
 var unsealed = {};
-Object.keys(unsealed).forEach(k => {
-  (k: number); // error: number ~> string
-});
+Object
+  .keys(unsealed)
+  .forEach(k => {
+    (k: number); // error: number ~> string
+  });
```

Comment:
I'm really not sure how to deal with `Object.keys`. `Observable.of` is another case.

I can't find a good heuristic to figure out when things should break:

```js
item
  .map(...)
  .filter(...);
```

vs

```js
Object.keys(...)
  .map(...)
  .filter(...);
```

The ast look exactly the same.

The only observable difference is the fact that the identifier is capitalized and the inside of .keys(...) is not a function.